api/src/main/java/org/apache/cloudstack/api/command/admin/systemvm/StopSystemVmCmd.java

```diff
@@ -54,7 +54,7 @@ public class StopSystemVmCmd extends BaseAsyncCmd {
                description = "The ID of the system virtual machine")
     private Long id;
 
-    @Parameter(name = ApiConstants.FORCED, type = CommandType.BOOLEAN, required = false, description = "Force stop the VM.  The caller knows the VM is stopped.")
+    @Parameter(name = ApiConstants.FORCED, type = CommandType.BOOLEAN, required = false, description = "Force stop the VM.  To be used if the caller knows the VM is stopped and should be marked as such.")
```

Comment:
```suggestion
    @Parameter(name = ApiConstants.FORCED, type = CommandType.BOOLEAN, required = false, description = "Force stop the VM.  (vm is marked as Stopped even when command fails to be send to the backend, otherwise a force poweroff is attempted). To be used if the caller knows the VM is stopped and should be marked as such.")
```